### PR TITLE
replaced RuntimeException with new MemberNotFoundException

### PR DESCRIPTION
**Pull request by ipinak**

### CODE REVIEW

File: member-service/src/main/java/com/hedvig/memberservice/aggregates/exceptions/MemberNotFoundException.java [hunk 1]
```diff
@@ -0,0 +1,7 @@
+package com.hedvig.memberservice.aggregates.exceptions;
+
+public class MemberNotFoundException extends Exception {
```
Comment (FredrikAreschoug) [inline member-service/src/main/java/com/hedvig/memberservice/aggregates/exceptions/MemberNotFoundException.java]:
```suggestion
public class MemberNotFoundException extends RuntimeException {
```

Comment (FredrikAreschoug) [reply member-service/src/main/java/com/hedvig/memberservice/aggregates/exceptions/MemberNotFoundException.java]: will probably also fix the tests :)

File: member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java [hunk 2]
```diff
@@ -188,7 +189,7 @@ void inactivateMember(InactivateMemberCommand command) {
       apply(new MemberInactivatedEvent(this.id));
     } else {
       String str =
-        String.format("Cannot INACTIAVTE member %s in status: %s", this.id, this.status.name());
+        String.format("Cannot DEACTIVATE member %s in status: %s", this.id, this.status.name());
```
Comment (FredrikAreschoug) [inline member-service/src/main/java/com/hedvig/memberservice/aggregates/MemberAggregate.java]: Just following the event naming, but yes it should be `MemberDeactivatedEvent`

**Comment by ipinak** [comment]
As I thought this is not the problem :(

**Pull request closed without merging**